### PR TITLE
refactor: refactor code and file-name

### PR DESCRIPTION
**Pull request by arminghm**
- update project dependencies
- now the service files include `.service` ext on filename
- change filename from camelCase to kebabCase
- remove socket code
- remove string.js file
- move StateService to `storage` directory
- move DATABASE_COLUMNS to StorageKey file

### CODE REVIEW

File: src/extension.ts [hunk 1]
```diff
@@ -1,71 +1,27 @@
...
 const apiEndPoint = 'http://localhost:1337';
-const socketEndPoint = 'http://localhost:1337';
 
-let socket: any = null;
 let state: StateService;
 let notificationService: NotificationService;
 let projectDataTreeProvider: ProjectDataTreeProvider;
 let signInTreeProvider: SignInDataTreeProvider;
 
-function initSocket() {
```
Comment (mohammadrafigh) [inline src/extension.ts:21]: Where is socket? we need socket integration in case user starts or stops an activity using other client apps.

Comment (arminghm) [reply src/extension.ts:21]: I'm going to postpone socket integration, as I remember there was problem for socket integration.

Comment (mohammadrafigh) [reply src/extension.ts:21]: Ok, But please add a warning in plugin description when adding plugin to vscode store about this. So users won't get confused with syncing problems between devices and clients.

**Review by mohammadrafigh — APPROVED** [review]

File: src/extension.ts (hunk 1 above)
Comment (arminghm) [reply src/extension.ts:21]: Sure

**Pull request merged**